<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>291</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="home">
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>231</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Choose Which Vending Machine To Use</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="coffee">
     <property name="styleSheet">
      <string notr="true">background-color:coral</string>
     </property>
     <widget class="QPushButton" name="coffee_drink">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Coffee</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hot_choc">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Hot Chocolate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hot_water">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>320</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Hot Water</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_btn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>450</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="cost_coffee">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_cost_coffee">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Cost</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="snack">
     <property name="styleSheet">
      <string notr="true">background-color:dodgerblue</string>
     </property>
     <widget class="QPushButton" name="MM">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>M&amp;M</string>
      </property>
     </widget>
     <widget class="QPushButton" name="crisps">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Crisps</string>
      </property>
     </widget>
     <widget class="QPushButton" name="snickers">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>260</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Snickers</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pantera_rosa">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>370</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Pantera Rosa</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="cost_snack">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_cost_snack">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Cost</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backsnack_btn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="drink">
     <property name="styleSheet">
      <string notr="true">background-color:greenyellow</string>
     </property>
     <widget class="QPushButton" name="coke">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>101</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Coke</string>
      </property>
     </widget>
     <widget class="QPushButton" name="water">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>270</y>
        <width>101</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Water</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="cost_drink">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_cost_drink_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Cost</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backdrink_btn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="coffee_btn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Coffee Machine</string>
    </property>
   </widget>
   <widget class="QPushButton" name="snack_btn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Snack Machine</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drink_btn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Drink Machine</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="total_money">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="fiftybtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>$0.50</string>
    </property>
   </widget>
   <widget class="QPushButton" name="onebtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>$1.00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twentybtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>$0.20</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fivebtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>$0.05</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tenbtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>$0.10</string>
    </property>
   </widget>
   <widget class="QLabel" name="total_change">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Total</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
